h/static/styles/excerpt.scss

```diff
@@ -0,0 +1,31 @@
+@import "compass/css3/images";
+@import "variables";
```

Comment:
IIRC this will pull in a second copy of 'variables.scss' rather than require them once. I want to use a system in future where requiring in CSS works the same way as JS modules.